proto/viam/component/camera/v1/camera.proto

```diff
@@ -104,12 +104,16 @@ message Webcams {
 }
 
 message Webcam {
-  // Camera driver label
+  // Camera driver label (for internal use only)
   string label = 1;
   // Camera driver status
   string status = 2;
   // Camera properties
   repeated Property properties = 3;
+  // Camera human-readable driver name
+  string name = 4;
+  // Camera unique identifier
+  string id = 5;
```

Comment:
maybe just unique_id. im assuming this value is permanent to the hardware and not just a bus? if it's not permanent, mabne we should comment that

Reply:
It depends. As is, on Linux this will be the bus info and on Mac this is unique to each device. So it may be permanent or may not be. We could change this in the future too but as of now I'm just following the Linux Kernel docs for Linux and Martha followed AVFoundation's recommendations. I don't think it really matters what we use since it's only for disambiguating identical camera names. We'll still use the label for everything else.

Reply:
sound sgood!